### PR TITLE
fix: eslint crashing

### PR DESCRIPTION
**Pull request by O4epegb**
## What does this PR do?

`yarn lint` crashes for me locally (and on CI too apparently, e.g. other random PR https://github.com/calcom/cal.com/actions/runs/5718689486/job/15495018755?pr=10488#step:5:18) because in App.tsx one of the type has the same name as App component, both `App`, basically shadowing
It's actually valid TS code, but it seems it requires some package updates to parse without errors, and since right now it's hard to update package versions and yarn.lock correctly, I decided to just fix this one line.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

Run `yarn lint`, it should not crash

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: apps/web/components/apps/App.tsx [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
 import type { UserAdminTeams } from "@calcom/features/ee/teams/lib/getUserAdminTeams";
 import Shell from "@calcom/features/shell/Shell";
 import classNames from "@calcom/lib/classNames";
+import { CAL_URL } from "@calcom/lib/constants";
 import { APP_NAME, COMPANY_NAME, SUPPORT_MAIL_ADDRESS } from "@calcom/lib/constants";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
+import type { RouterOutputs } from "@calcom/trpc/react";
 import { trpc } from "@calcom/trpc/react";
-import type { App as AppType } from "@calcom/types/App";
+import type { App as AppType, AppFrontendPayload } from "@calcom/types/App";
```
Comment (O4epegb) [inline apps/web/components/apps/App.tsx:18]: This is the only change basically, everything else is autoformat etc

**Comment by O4epegb** [comment]
And now after the fix there are 55 eslint-prettier errors, should I fix them here too? Or make them warnings?

**Comment by nicktrn** [comment]
Can't help here but came to say well done! Saw that error before myself but never got the full call stack, so decided to ignore it in the end. Others have had the problem too, see #10292.

**Review by emrysal — DISMISSED** [review]
Amazing fix! Turns out our lint:report command has been failing silently and our lint analyse actually succeeding, so the errors that you found made it in over the last week.

lgtm

**Review by emrysal — APPROVED** [review]

**Pull request merged**